review/fileToReview.js

```diff
@@ -12,62 +17,42 @@ var domain = "bank.local.fr"
  * @return {Object} All transactions available on the page
  */
 async function fetchTransactions(fromDate, authorization, jws = null, id, page, previousTransactions) {
```

Comment:
Optional parameters must be specified at the end of the function definition. 
If you don't have the JWS parameter you must put NULL yourself in this case